### DISCUSSION TITLE
How to use aliases?

### DISCUSSION THREAD

**Discussion by ghost** [Intevel/nuxt-directus#103] [Q&A] [answered]
I want to retrieve the `meta_description` field as `description`. According to [directus docs](https://docs.directus.io/reference/query.html#aliases) it should work somehow like this:

```js
const { data, error } = await useAsyncData('about', () => getSingletonItem({
    collection: 'about',
    params: {
        fields: ['*', 'meta_description'],
        alias: '[description]=meta_description'
    }
}))
```

… but it doesn’t work …

How to use aliases?

**Comment by rijkvanzanten** [comment, marked as answer]
Close! The way to translate query params to JS params object is slightly different:

```js
const { data, error } = await useAsyncData('about', () => getSingletonItem({
    collection: 'about',
    params: {
        'alias[description]': 'meta_description'
    }
}))
```

(you can omit fields here, as it defaults to `*`, and `*` already includes `meta_description` as well)

**Reply by ghost** [reply]
@rijkvanzanten Nice, the one variation I did not try ;)

Thanks!

**Reply by ghost** [reply]
Btw you’ve forgot the second quotation mark like
```
'alias[description]': 'meta_description'
                   ↑
```

**Reply by rijkvanzanten** [reply]
Ty! Updated my example
